pkg/cli/session/session.go

```diff
@@ -312,6 +328,24 @@ set -euo pipefail
 	if err != nil {
 		return err
 	}
+	if e.Options.ManagementClusterID != "" {
+		// This add a command to access management cluster
+		err = e.createBin("oc-mc", "oc --kubeconfig="+filepath.Join(e.Path, e.Options.ManagementClusterID, "config")+" $@")
```

Comment:
Shall we update session help text to show how to use `oc-mc` and `oc-sc` bins ? 

Ref : https://github.com/openshift/backplane-cli/blob/6411c2f1a7c07b07c538b15b76e423d2d8430760/pkg/cli/session/session.go#L417